### PR TITLE
feat(tutorial): complete step 2

### PR DESCRIPTION
**Pull request by marklindsay24**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/content/LandingPage/LandingPage.js [hunk 1]
```diff
@@ -1,7 +1,98 @@
...
+      <div className="bx--row landing-page__banner">
+        <Breadcrumb noTrailingSlash aria-label="Page navigation">
+          <BreadcrumbItem>
+            <a href="/">Getting started</a>
+          </BreadcrumbItem>
+        </Breadcrumb>
+        <h1 className="landing-page__heading">
+          Design &amp; build with Carbon
+        </h1>
+      </div>
```
Comment (kellychurchill) [inline src/content/LandingPage/LandingPage.js]: Your Getting Started and Design & Build with Carbon heading are all on the same line.

Comment (marklindsay24) [reply src/content/LandingPage/LandingPage.js]: How do I recover step 2, to resolve and re-submit? I did have problems before with recovering my step 2.

Comment (kellychurchill) [reply src/content/LandingPage/LandingPage.js]: You need to check out the branch. 

See
https://www.atlassian.com/git/tutorials/using-branches/git-checkout

Comment (marklindsay24) [reply src/content/LandingPage/LandingPage.js]: I know how to do that (checkout), but for some reason I have 1 uncommited change to upstream, and do not know how to clear it out. 
 I have tried 'git reset --hard '

Not sure how to not have it attempt to push upstream.

Comment (kellychurchill) [reply src/content/LandingPage/LandingPage.js]: You cannot push to upstream as that is the carbon-design-system repo. You must push to origin. 

See https://www.carbondesignsystem.com/tutorial/react/step-2#git-commit-and-push

File: src/content/RepoPage/RepoPage.js [hunk 2]
```diff
@@ -1,7 +1,70 @@
...
+    stars: '456',
+    links: 'Links',
+  },
+];
 
 const RepoPage = () => {
-  return <div>REPO PAGE</div>;
+  return (
+    <div className="bx--grid bx--grid--full-width bx--grid--no-gutter repo-page">
+      <div className="bx--row repo-page__r1">
```
[the remark this answers is not in the document]
Comment (marklindsay24) [reply src/content/RepoPage/RepoPage.js]: If I have to re-do step two all over again, just to correct a simple cut & paste job from tutorial, I will. However, I think I've spent more time trying to figure out how to get back to my step2 without git trying to push to master/upstream.

Here is the corrected content:
return (
  <div className="bx--grid bx--grid--full-width bx--grid--no-gutter repo-page">
    <div className="bx--row repo-page__r1">
      <div className="bx--col-lg-16">Data table will go here</div>
    </div>
  </div>
);

As I've mentioned in separate email. I have designed a whole web portal for Resiliency using carbon and reach. I believe I have the skills. I can give a demo of such portal (https://clientconnect.mybluemix.net/).

[the remark this answers is not in the document]
Comment (marklindsay24) [reply src/content/RepoPage/RepoPage.js]: `return (
  <div className="bx--grid bx--grid--full-width bx--grid--no-gutter repo-page">
    <div className="bx--row repo-page__r1">
      <div className="bx--col-lg-16">Data table will go here</div>
    </div>
  </div>
);`

[the remark this answers is not in the document]
Comment (marklindsay24) [reply src/content/RepoPage/RepoPage.js]: `
return (
  <div className="bx--grid bx--grid--full-width bx--grid--no-gutter repo-page">
    <div className="bx--row repo-page__r1">
      <div className="bx--col-lg-16">Data table will go here</div>
    </div>
  </div>
);
`

File: src/content/LandingPage/LandingPage.js (hunk 1 above)
Comment (marklindsay24) [reply src/content/LandingPage/LandingPage.js]: See forced push...but not sure that worked. I seem to really have messed up my git repo with force option.

**Comment by marklindsay24** [comment]
Much difficulty in using git trying to get back to a working step 2 so that I can commit without it trying to pull from upstream.
As result, i just started step 2 again and did another pull request. Please close out my completion of this tutorial via this pull (for step2):

https://github.com/carbon-design-system/carbon-tutorial/pull/2993

**Pull request closed without merging**